addons/point_of_sale/views/pos_order_view.xml

```diff
@@ -48,6 +48,7 @@
                                 <field name="full_product_name"/>
                                 <field name="pack_lot_ids" widget="many2many_tags" groups="stock.group_production_lot"/>
                                 <field name="qty"/>
+                                <field name="customer_note" optional="hide" widget="text"/>
```

Comment:
No need of widget="text" as it is a Char field

Reply:
done